server/proxy.go

```diff
@@ -208,6 +208,9 @@ func (p *Proxy) addHeaders(sessionClaims samlsp.JWTSessionClaims, headers http.H
 	if p.config.AttributeHeaderWildcard != "" {
 		for attr, values := range sessionClaims.GetAttributes() {
 			for _, value := range values {
+				if uri, err := url.Parse(attr); err == nil {
+					attr = strings.Trim(strings.Replace(uri.Path, "/", "-", -1), "-")
```

Comment:
This seems reasonable. Can you add a brief description of this to the usage text in the Config struct?

Reply:
Sure